deploy/crds/apps.3scale.net_v1alpha1_apimanager_cr_monitoring.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: example-apimanager-s3
```

Comment:
Is this example name intentional or should we choose another one?

Reply:
other one